src/components/general/Menu/__stories__/Menu.stories.tsx

```diff
@@ -17,29 +38,23 @@ const MenuStory = () => {
                 sections={[
                     {
                         key: 'This is the only section, but I still need a key',
-                        items: [
-                            {
-                                key: '1',
-                                title: 'First',
-                            },
-                            {
-                                key: '2',
-                                title: 'Selected',
-                                isSelected: true,
-                            },
-                            {
-                                key: '3',
-                                title: 'Disabled',
-                                isDisabled: true,
-                            },
-                            {
-                                key: '4',
-                                title: 'Last',
-                            },
-                        ],
+                        items: items,
                     },
                 ]}
             />
+            <div style={{width: '300px'}}>

```

Comment:
I've added a "Narrow version" of the drop down here. I haven't seen a lot of text explaining how or why in the Storybook so it's just the component wrapped in a div with a set width. What do you think @MartinForre? I've done the same thing with the menu component 😄